src/components/faceVerification/screens/IntroScreen.jsx

```diff
@@ -194,6 +241,26 @@ const IntroScreen = ({ styles, screenProps, navigation }) => {
     )
   }
 
+  if (disposing !== false) {
```

Comment:
what is this? disposing meaning the user is deleting his face record.

Reply:
removed